json-extension/src/common/index.ts

```diff
@@ -7,3 +7,6 @@
 
 export const JSON_LANGUAGE_ID = 'json';
 export const JSON_LANGUAGE_NAME = 'JSON';
+
+export const JSONC_LANGUAGE_ID = 'jsonc';
```

Comment:
(nit) it would be nice to replace other occurrences of using `jsonc` as a language id with `JSONC_LANGUAGE_ID`, e.g. in monaco and textmate registrations.

Reply:
done.